CHANGELOG.md

```diff
@@ -1,13 +1,21 @@
 # Changelog
 
+## 0.31.4
+
+### Patch Changes
+
+- 63742da: - **feat**: added support for `EIP-5792` (https://eips.ethereum.org/EIPS/eip-5792) in `OnchainKitProvider`. By @0xAlec #1181
```

Comment:
nit: remove this commit hash in the beginning

Reply:
i will do that in the next PR